bridge/deploy_bridge_and_du2.sh

```diff
@@ -7,31 +7,39 @@ cd `dirname $0`
 CONTRACTS=streamr/tokenbridge-contracts
 #CONTRACTS=poanetwork/tokenbridge-contracts
 
-AMBTASK="deploy_amb"
 AMBRESULTS="bridgeDeploymentResultsAMB.json"
-
-ERC677TASK="deploy_erc677"
 ERC677RESULTS="bridgeDeploymentResultsERC677.json"
 
 echo "1. Deploying AMB"
-docker run --name $AMBTASK --env-file amb.env $CONTRACTS deploy.sh
-docker cp $AMBTASK:/contracts/deploy/bridgeDeploymentResults.json $AMBRESULTS
+TASK=amb
```

Comment:
Task name changed. Is this a change in xdai images?

Reply:
it's just the name used to identify the docker process